pkgs/top-level/python-aliases.nix

```diff
@@ -792,6 +792,7 @@ mapAliases ({
   types-enum34 = throw "types-enum34 is obselete since Python 3.4"; # added 2025-02-15
   types-paramiko = throw "types-paramiko has been removed because it was unused."; # added 2022-05-30
   types-typed-ast = throw "types-typed-ast was removed because so was typed-ast"; # added 2025-05-24
+  uamqp = throw "'uamqp' has been removed because it is broken and unmaintained."; # added 2025-06-11
```

Comment:
Unmaintained? It had a commit two days ago.
Maybe you should open an upstream issue about GCC 14.

Reply:
Never mind. It is in fact unmaintained and I doubt they would fix any issues.